apollo-router/tests/samples/basic/interface-object/plan.json

```diff
@@ -201,15 +201,15 @@
                         {
                             "request": {
                                 "body": {
-                                    "query": "query TestItf2__reviews__2($representations:[_Any!]!){_entities(representations:$representations){...on C{c}}}",
+                                    "query": "query TestItf2__reviews__2($representations: [_Any!]!) { _entities(representations: $representations) { ... on C { c } } }",
                                     "operationName": "TestItf2__reviews__2",
                                     "variables": {
                                         "representations": [
                                             {
                                                 "__typename": "C",
                                                 "i": {
                                                     "x": 1,
-                                                    "y": 1
+                                                    "y": null
```

Comment:
The subgraph request sent by Router is now:

```json
{"query":"query TestItf2__reviews__2($representations: [_Any!]!) { _entities(representations: $representations) { ... on C { c } } }","operationName":"TestItf2__reviews__2","variables":{"representations":[{"__typename":"C","i":{"x":1,"y":null},"id":"1"}]}}
```

… but I don’t know why that `null` is there, since the previous mock returns `{"data":{"_entities":[{"y":1}]}}`